web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -149,8 +155,13 @@ function ProjectTopSection({
                 mt: 2.5,
               }}
             >
-              {!metadata && loading && <Skeleton variant="text" height={124} />}
-              {!onChainProject && (
+              {onChainProjectId ? (
+                projectName && (
```

Comment:
if this is on-chain, show projectId label IFF projectName is displayed above. 

If projectName is not displayed above, the header falls back to `Project ${onChainProjectId}`. We don't want to display projectId here again in this case.